starwars/src/components/StarWarsData.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './StarWars.css';
+
+const Makers = props => {
+    return(
+        <div className="makers-container">
+            <h1 class="makers-name">Name:{props.Makers.name}</h1>
+            <p class="makers-birth_year">Birth year:{props.Makers.birth_year}</p>
+            <p class="makers-height">Height:{props.Makers.height}</p>
+            <p class="makers-gender">Gender:{props.Makers.gender}</p>
+            <p class="makers-eye_color">Eye color:{props.Makers.eye_color}</p>
+        </div>
+    )
+}
```

Comment:
Again, this is your final functional component. For convention and readability, you should name your function "Makers" as "StarWarsData", and whatever naming convention you used when you passed the props will replace "Makers" , a suggestion would be "charsInfo", so it would be props.charsInfo.name, props.charsInfo.birth_year ... and so forth down the line

Reply:
dont forget to export correct function after  you have renamed